server/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/Db2SqlParser.g4

```diff
@@ -1556,7 +1556,7 @@ dbs_seclabel_name: IDENTIFIER {validateLength($IDENTIFIER.text, "security label"
 dbs_sequence_name: T=dbs_sql_identifier {validateLength($T.text, "sequence name", 128);};
 dbs_servauth_value: NONNUMERICLITERAL;
 dbs_simple_when_clause: (WHEN (dbs_basic_predicate | dbs_expressions) THEN (dbs_result_expression1 | NULL))+;
-dbs_smallint: T=dbs_integer_constant {if(!(Integer.parseInt($T.text) > -2 && Integer.parseInt($T.text) < 100)) { notifyError("paser.validValueMsg", $T.text, "in range -1 to 99");}};//MINUSCHAR? SINGLEDIGITLITERAL SINGLEDIGITLITERAL?;// java ref - -1 to 99
+dbs_smallint: T=dbs_integer_constant {validate99($T.text);};//MINUSCHAR? SINGLEDIGITLITERAL SINGLEDIGITLITERAL?;// java ref - -1 to 99
```

Comment:
The name is not clear, maybe something `validateTextInRange` and provide the range.